styles/style.css

```diff
@@ -0,0 +1,504 @@
+  padding-left: 200px;
+}
+
+.quote {
+  position: relative;
+}
+
+span {
+  line-height: 1.125rem;
+  display: block;
+  position: relative;
+  left: 0;
+}
+
+.main-article {
+  width: 54%;
+  display: block;
+  padding-top: 40px;
+  margin-left: 112px;
+}
+
+.author {
+  color: rgb(0, 0, 0);
+font-family: Verdana, Geneva, Tahoma, sans-serif;
+text-decoration: none;
+  font-size: 100%;
+  font-weight: 500;
+  height: 23px;
+  line-height: 21px;
+}
+
+.author-link:hover{
+  text-decoration: underline;
+  color: gray;
+}
+
+.section-container {
+  margin-top: 60px;
+}
+.left-section {
+  float: left;
+}
+
+li {
+  display: inline;
+}
+
+.social > li {
+  float: right;
+  margin-right: 20px;
+}
+
+.social-icon {
+  width: 15px;
+  height: 15px;
+}
+
+.right-section {
+  padding-right: 8px;
+  float: right;
+}
+
+.main-text-container {
+  margin-top: 20px;
+}
+
+.main-text {
+  margin-top: 20px;
+  font-family: nyt-imperial,georgia,'times new roman',times,serif;
+  display: block;
+  font-size: 1.25rem;
+  line-height: 1.875rem;
+}
+
+.caption-text {
+  font-size: 0.875rem;
+  line-height: 1.125rem;
+  font-weight: 400;
+  font-style: normal;
+  color: rgb(102, 102, 102);
+}
+
+.images {
+  margin-top: 30px;
+}
+
+.pict-img {
+  width: 80%;
+}
+
+.aside-scene > ul > li {
+  float: left;
+  width: 25%;
+  padding: 5px;
+  font-family: nyt-franklin, arial, helvetica, sans-serif;
+  font-size: 11px;
+  line-height: 17px;
+  font-weight: 300;
+  color: #000000;
+}
+
+.scene-1 {
+  width: 100%;
+  height: 190px;
+  background-size: cover;
+  position: relative;
+  background: url(../img/inflation.png) 0px 0px;
+  background-size: 1050px 800px;
+}
+
+.scene-2 {
+  width: 100%;
+  height: 190px;
+  background-size: cover;
+  position: relative;
+  background: url(../img/inflation.png) -244px 0px;
+  background-size: 1050px 800px;
+}
+
+.scene-3 {
+  width: 100%;
+  height: 190px;
+  background-size: cover;
+  position: relative;
+  background: url(../img/inflation.png) -504px 0px;
+  background-size: 1050px 800px;
+}
+
+.scene-4 {
+  width: 100%;
+  height: 190px;
+  background-size: cover;
+  position: relative;
+  background: url(../img/inflation.png) -777px 0px;
+  background-size: 1050px 800px;
+}
+
+.scene-5 {
+  width: 100%;
+  height: 190px;
+  background-size: cover;
+  position: relative;
+  background: url(../img/inflation.png) 0px -1220px;
+  background-size: 1050px 800px;
+}
+
+.scene-6 {
+  width: 100%;
+  height: 190px;
+  background-size: cover;
+  position: relative;
+  background: url(../img/inflation.png) -250px -1200px;
+  background-size: 1050px 800px;
+}
+
+.scene-7 {
+  width: 100%;
+  height: 190px;
+  background-size: cover;
+  position: relative;
+  background: url(../img/inflation.png) -515px -1200px;
+  background-size: 1050px 800px;
+}
+
+.scene-8 {
+  width: 100%;
+  height: 190px;
+  background-size: cover;
+  position: relative;
+  background: url(../img/inflation.png) -785px -1200px;
+  background-size: 1050px 800px;
+}
```

Comment:
There are lot of repetition. maybe you can divide them into separate classes to reduce the repetition. Find the common pattern that are occurring again and again. Enclose them inside a class and use it whenever you need it.